openpgp/add.go

```diff
@@ -255,66 +256,31 @@ func (w *Worker) UpdateKey(pubkey *Pubkey) error {
 	err = pubkey.Visit(func(rec PacketRecord) (err error) {
 		switch r := rec.(type) {
 		case *Pubkey:
-			_, err := Execv(tx, `
-UPDATE openpgp_pubkey SET
-	creation = $2, expiration = $3, state = $4, packet = $5,
-	ctime = $6, mtime = $7,	md5 = $8, sha256 = $9,
-	algorithm = $10, bit_len = $11, unsupp = $12
-WHERE uuid = $1`, r.RFingerprint,
-				r.Creation, r.Expiration, r.State, r.Packet,
-				r.Ctime, r.Mtime, r.Md5, r.Sha256,
-				r.Algorithm, r.BitLen, r.Unsupported)
+		 	err := updater.UpdatePubkey(r)
```

Comment:
Now that these update methods are short enough, they're all good candidates for rewriting to:

```go
if err := updater.XYZ(a, b); err != nil {
  return err
}
```